docs/jwst/outlier_detection/outlier_detection_tso.rst

```diff
@@ -0,0 +1,41 @@
+.. _outlier-detection-tso:
+
+Outlier Detection for TSO Data
+==============================
+
+Time-series observations (TSO) result in input data stored as a 3D CubeModel
+where each plane in the cube represents a separate integration without changing the
+pointing.  Normal imaging data benefit from combining all integrations into a
+single image. TSO data's value, however, comes from looking for variations from one
+integration to the next.  The outlier detection algorithm, therefore, gets run with 
+a few variations to accomodate the nature of these 3D data.
+
+#. Input data is converted into a CubeModel (3D data array) if a ModelContainer
+   of 2D data arrays is provided.
+
+#. The median image is created without resampling the input data
+
+   * The median image is created by combining all planes in the 
+     ModelContainer pixel-by-pixel using a rolling-median algorithm, in order
+     to flag outliers frame-by-frame but preserve real time variability.
```

Comment:
Oooh, we don't want to overload the use of the term "frame" here. A "frame" is a very specific thing in JWST data and refers to the data from a single readout of a detector. I believe you want to say "integration-by-integration" here.